api-dump/AWSDataStorePlugin.json

```diff
@@ -8205,7 +8205,7 @@
       "-module",
       "AWSDataStorePlugin",
       "-o",
-      "\/var\/folders\/hn\/5bx1f4_d4ds5vhwhkxc7vdcr0000gn\/T\/tmp.nAGRifhwH6\/AWSDataStorePlugin.json",
+      "\/var\/folders\/hw\/1f0gcr8d6kn9ms0_wn0_57qc0000gn\/T\/tmp.Op48snuvLw\/AWSDataStorePlugin.json",
```

Comment:
is this change part of the PR?

Reply:
I think this is updating the api dumps because of the changed protocol for KeychainStore (the _getAll() function)? In that case this should be part of the PR